logstage/logstage-api-logger/src/main/scala/com/github/pshirshov/izumi/logstage/api/rendering/StringRenderingPolicy.scala

```diff
@@ -202,5 +205,12 @@ object StringRenderingPolicy {
     override val isSpace: Boolean = false
   }
 
-
+  val configPolicyMapper: RenderingPolicyMapper[StringRenderingPolicy] = new RenderingPolicyMapper[StringRenderingPolicy] {
+    override def instantiate(config: Config): StringRenderingPolicy = {
```

Comment:
Could you remove dependency on typesafe-config from here by reading from a case class instead?

```
final case class PolicyConfig(withColors: Boolean, withExceptions: Boolean, layout: String)

def instantiate(pol: PolicyConfig) = new StringRenderingPolicy(options = RenderingOptions(pol.withColors, pol.withExceptions), renderingLayout = pol.layout)
```

and in `logstage-di` add:

```
make[StringRenderingPolicy].from { 
  pol: PolicyConfig @ConfPath("logstage.rendering-policy") =>
    StringRenderingPolicy.instantiate(PolicyConfig)
}
```